jlox/app/src/main/java/dev/zxul767/lox/runtime/Interpreter.java

```diff
@@ -312,12 +316,12 @@ public Object visitCallExpr(Expr.Call expr) {
       );
     }
     LoxCallable function = (LoxCallable)callee;
-    if (args.size() != function.arity()) {
+    if (args.size() != function.signature().arity()) {
```

Comment:
Since `LoxCallable` (an interface) requires `signature: CallableSignature`, it seems like it would be redundant. So I'm inclined to say no now, but I'll think about it more...